amnestywebsite/humanity-theme
# Add FSE PDF template

Ref: https://github.com/amnestywebsite/humanity-theme/issues/246

Current-language document template has been created as an FSE template in the theme, rather than a PHP template in the plugin.

Remote document template is still loaded as PHP via the SP runner plugin in the short term. In order to port that template over into the theme, we'd need to create many new blocks, and is a task for the future.

**Steps to test**:
1. requires SP runner plugin
2. view a document (via /latest/research/)
3. the document should load correctly
